test/src/e2e_vm_tests/test_programs/should_pass/language/self_impl_reassignment/src/main.sw

```diff
@@ -39,10 +39,7 @@ impl E {
 }
 
 fn main() -> bool {
-    let mut a = A {
-        a: 0,
-        b: 0,
-    };
+    let mut a = A { a: 0, b: 0 };
```

Comment:
I ran `forc fmt` on this so that it won't change in the future. If the line numbers change, tests will break.